lookout/style/format/benchmarks/top_repos_quality.py

```diff
@@ -157,7 +157,7 @@ def __init__(self, quality: Optional[str] = None, model: Optional[str] = None) -
 
 
 def measure_quality(repository: str, from_commit: str, to_commit: str, port: int,
-                    review_config: dict, train_config: dict) -> QualityReport:
+                    review_config: dict, train_config: dict, bblfsh: str=None) -> QualityReport:
```

Comment:
Type annotation should be `Optional[str]` and we should probably remove the default value to make sure no part of the codebase is mistakingly forgetting to pass on bblfsh arg (now or in the future).